src/ChangeStream.php

```diff
@@ -42,6 +42,9 @@ class ChangeStream implements Iterator
      */
     const CURSOR_NOT_FOUND = 43;
 
+    /** @var int */
+    private static $cursorNotFound = 43;
```

Comment:
There's `CURSOR_NOT_FOUND` a couple of lines above that could be used. Alternatively, I'm fine deprecating the public constant in favour of a private one (class constant visibility will be available to us once we drop PHP 7.0, so we'd continue using `private static` until then).

Reply:
Indeed. I didn't want to use the `CURSOR_NOT_FOUND` because it's already deprecated (at least in documentation). Even if we dropped PHP 7.0 before PHPLIB 2.0, we couldn't change `CURSOR_NOT_FOUND` to `private` without it being a BC break.